docker/gaffer-operation-runner/entrypoint.sh

```diff
@@ -11,9 +25,9 @@ fi
 echo "Using Graph Id: ${graph_id}"
 # Find all the Jars and produce comma seperated list
 jars=()
-while IFS=  read -r -d $'\0'; do
-    jars+=("$REPLY")
-done < <(find /gaffer/jars -name "*.jar" -print0)
+for jar in $(find /gaffer/jars -name "*.jar"); do
+  jars+=${jar}
+done
```

Comment:
This change doesn't work for me - I end up with a concatenated string with no delimiter, whereas before it gave a CSV

Reply:
Ah yeah good spot, I forgot the brackets around the ${jar} variable